### ISSUE TITLE
[Accessibility]: The <label> element does not have descriptive text that identifies the expected input

### ISSUE THREAD

**Issue by yashgarg1719** [IBM/carbon-components-angular#2195]
## Detailed description

> Describe in detail the issue you're having.

This issue is specific to the `<label>` element inside the `<ibm-toggle>` component, which does not have descriptive text that identifies the expected input.

> Is this a feature request (new component, new icon), a bug, or a general issue?

Bug

> Is this issue related to a specific component?

`<ibm-toggle>`

> What did you expect to happen? What happened instead? What would you like to see changed?

A `<label>` element must have non-empty descriptive text that identifies the purpose of the interactive component

> What browser are you working in?

Firefox 101.0(64-bit)

> What version of the Carbon Design System are you using?

`carbon-components@10.5.0`
`carbon-components-angular@4.56.4`

> What WCAG 2.1 checkpoint does the issue violate?

IBM 2.4.6 Headings and Labels WCAG technique G131

## Additional information

* Screenshots or code

![image](https://user-images.githubusercontent.com/108860673/178186556-97f27259-ccc7-4dac-8037-1f0518f38946.png)

**Comment by Akshat55** [comment]
What `description` should it have? I've ran the accessibility checker on this component and did not find such error. Could you please create a code sandbox and provide more issue for this issue.
